### PR TITLE
[StatusBar] add media controller panel

### PR DESCRIPTION
**Pull request by neochapay**

### CODE REVIEW

File: src/qml/AppLauncher.qml [hunk 1]
```diff
@@ -34,7 +34,7 @@ import QtQuick.Controls.Styles.Nemo 1.0
 
 GridView {
     id: gridview
-    width: cellWidth * columns
```
Comment (eekkelund) [inline src/qml/AppLauncher.qml]: Why is this change needed?

Comment (neochapay) [reply src/qml/AppLauncher.qml]: Removes unnecessary calculation

Comment (eekkelund) [reply src/qml/AppLauncher.qml]: But GridView doens't necessarily have same width as parent Page.

File: src/qml/statusbar/MediaController.qml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                    rightMargin: width/2
+                    verticalCenter: playPause.verticalCenter
+                }
+
+                MouseArea{
+                    anchors.fill: parent
+                    onClicked: if (mprisManager.canGoPrevious) mprisManager.previous()
+                }
+
+                source: "/usr/share/themes/glacier/fontawesome/icons/backward.png"
```
Comment (eekkelund) [inline src/qml/statusbar/MediaController.qml]: "image://theme/backward" Should do the trick. :)
Same with others.

Comment (neochapay) [reply src/qml/statusbar/MediaController.qml]: image://theme/glacier/fontawesome ?

Comment (eekkelund) [reply src/qml/statusbar/MediaController.qml]: It searches default from that folder:
https://github.com/nemomobile/qtquickcontrols-nemo/blob/master/src/controls/themedaemon/mlocalthemedaemonclient.cpp#L114
So it works like this:
https://github.com/nemomobile/qtquickcontrols-nemo/blob/master/examples/touch/content/IconPage.qml#L48
Try if it works with "image://theme/backward"

File: src/qml/Statusbar.qml [hunk 3]
```diff
@@ -289,9 +291,18 @@ Item {
             iconSize: Theme.itemHeightExtraSmall
             source: "image://theme/icon_gps_normal"
         }
+
         StatusbarItem {
             iconSize: Theme.itemHeightExtraSmall
-            source: "image://theme/icon_play_pause"
+            source: (mprisManager.currentService && mprisManager.playbackStatus == Mpris.Playing) ?
+                        "/usr/share/themes/glacier/fontawesome/icons/pause.png"
+                      : "/usr/share/themes/glacier/fontawesome/icons/play.png"
```
Comment (eekkelund) [inline src/qml/Statusbar.qml]: Here same with image://theme...
I didn't yet have time to test this PR, did you @locusf manage?

Comment (neochapay) [reply src/qml/Statusbar.qml]: Doh....i fix it evening sorry

Comment (eekkelund) [reply src/qml/Statusbar.qml]: NP:)
How to try this PR? Install glacier-music and have some songs?
I will try today evening or tomorrow.

Comment (neochapay) [reply src/qml/Statusbar.qml]: Yeap :)

Comment (locusf) [reply src/qml/Statusbar.qml]: Nope, didn't have time to test this yet.
